documentation/proc-pages/physics-models/fusion_reactions/plasma_reactions.md

```diff
@@ -0,0 +1,236 @@
+</figure>
+
+#### Attributes Updated
+The method updates the following attributes:
+
+- `f_dd_proton_branching (float)`: Branching ratio of tritium producing D-D reactions to 3He ones.
+
+-----------------------
+
+### Calculate fusion reactions
+
+There are 4 key functions for calculating the fusion reaction for the plasma. They are `dt_reaction()`, `dhe3_reaction()`, `dd_helion_reaction()` and `dd_triton_reaction()`. They all perform the same key calculations below but with their own specific values for their reactions.
+
+#### Detailed Steps
+1. **Initialize Bosch-Hale Constants**: Initializes the Bosch-Hale constants for the required reaction using predefined reaction constants stored in the BoschHaleConstants dataclass.
+2. **Calculate Fusion Reaction Rate**: Uses Simpson's rule to integrate the fusion reaction rate over the plasma profile.
+3. **Calculate Fusion Power Density**: Compute the fusion power density produced by the given reaction. Using the reaction energy calculated and stored in `constants.f90`. The reactant density is is given by $\mathtt{f\_deuterium, f\_tritium}$ or $\mathtt{f\_helium3}$ multiplied by the volume averaged ion density.
+4. **Calculate Fusion Power Densities**: Compute the fusion power density for alpha particles, neutrons and other charged particles, depending on the reaction. Energy branching fractions used are calculated and called from `constants.f90`
+5. **Calculate Fusion Rate Densities**: Compute the fusion rate density and for alpha particles, neutrons and other charged particles, depending on the reaction.
+6. **Update Reaction Power Density**: Updates the object attribute for the specific reaction power density.
+7. **Sum Fusion Rates**: Call the [`sum_fusion_rates()`](#sum-the-fusion-rates--sum_fusion_rates) function to add the reaction to the global plasma power balance.
+
+#### Attributes Updated
+The method updates the following attributes:
+
+- `self.sigmav_dt_average`: Average fusion reaction rate `<sigma v>` for D-T.
+- `self.dt_power_density`: Fusion power density produced by the D-T reaction.
+- `self.alpha_power_density`: Power density of alpha particles produced.
+- `self.charged_power_density`: Power density of charged particles produced.
+- `self.neutron_power_density`: Power density of neutrons produced.
+- `self.fusion_rate_density`: Fusion reaction rate density.
+- `self.alpha_rate_density`: Alpha particle production rate density.
+- `self.proton_rate_density`: Proton production rate density.
+
+-----------------------
+
+### Sum the fusion rates | `sum_fusion_rates()`
+
+This method updates the cumulative class fusion power densities and reaction rates for alpha particles, charged particles, neutrons, and protons.
+
+#### Parameters:
+- `alpha_power_add` (float): Alpha particle fusion power per unit volume [MW/m³].
+- `charged_power_add` (float): Other charged particle fusion power per unit volume [MW/m³].
```

Comment:
Only case where you have mentioned Other charged particle, is this the only case not including alphas in it or do all the others not include alphas too? If so may need to clarify other statements